src/client/js/script.js

```diff
@@ -62,5 +62,582 @@ $(document).ready(function() {
     $('#name,#message,#email').css('border-color','#484848');
     ctr = 0;
     validate_form();
-  }); 
+  });
+/*
```

Comment:
This is the cause of the error. You have deleted the closing brackets here. Please make the change.